src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -615,6 +615,10 @@ void AttitudeEstimatorQ::task_main()
 			ctrl_state.q[2] = _q(2);
 			ctrl_state.q[3] = _q(3);
 
+			ctrl_state.x_acc = _accel(0);
+			ctrl_state.y_acc = _accel(1);
+			ctrl_state.z_acc = _accel(2);
+
```

Comment:
@tumbili Just a suggestion - Wouldn't the position estimator have a much better estimate of the body acceleration. It should also be able to estimate the accel biases much better.